player.py

```diff
@@ -17,4 +17,24 @@ def main(self, display):
         pygame.draw.rect(display, (255, 0, 0), (self.x, self.y, self.width, self.height))
 
     def update_score(self, add_point=1):
-        self.score += add_point
\ No newline at end of file
+        self.score += add_point
+
+    #function to display the current score of the player on the game screen
```

Comment:
I think the function name is descriptive enough to tell us what it does, so there may be no need for this comment.  You could even call the function display_current_score.